### DISCUSSION TITLE
Input date doesn't accept string as input in ngModel

### DISCUSSION THREAD

**Discussion by Kuuotai** [taiga-family/taiga-ui#3678] [General]
The default output of tui-input-date is a string but the input is TuiDay, why not also accept string and convert it to TuiDay within the component? It's a hassle to always convert to that type of Object when dates are commonly used as string also.

**Comment by waterplea** [comment]
The default output is also `TuiDay`, not sure where you got the string, perhaps from listening to the native `input` tag? You can configure it to use any format with DI tokens, demo has an example for native `Date` object, but you can configure it to work with strings too:
https://taiga-ui.dev/components/input-date#native-date-output
